### PR TITLE
Time series docs

### PR DESCRIPTION
**Pull request by tmgordeeva**
Docs for time series aggregation.

### CODE REVIEW

**Review by martijnvg — COMMENTED** [review]
Like we discussed, for now let's just have a general description of the time series aggregation , description of the supported parameters and a sample usage. Let's not create a full doc test for now, since time_series agg is tech preview.

File: docs/reference/aggregations/bucket/time-series-aggregation.asciidoc [hunk 1]
```diff
@@ -0,0 +1,100 @@
+[[search-aggregations-bucket-time-series-aggregation]]
+=== Time Series Aggregation
+++++
+<titleabbrev>Time Series</titleabbrev>
+++++
+
+The time series aggregation queries data created using a time series index. This is typically data such as metrics
```
Comment (martijnvg) [inline docs/reference/aggregations/bucket/time-series-aggregation.asciidoc:7]: I think `preview::[]` should be added here to indicate that this feature is techpreview.

**Review by martijnvg — COMMENTED** [review]
This change looks good. Let's try to get the docs build to succeed and see how the new page looks in the generated preview.

**Comment by kilfoyle** [comment]
In case it helps, I think at line 92 there's an ID `search-aggregations-bucket-terms-aggregation-size` that's already in use, and should be changed to something like `search-aggregations-bucket-time-series-aggregation-size`.

**Review by martijnvg — APPROVED** [review]
Maybe remove `//////////////////////////` on line 98 and line 97, otherwise LGTM

**Review by kilfoyle — APPROVED** [review]
LGTM! 🚀

**Pull request merged**